pages/linux/chage.md

```diff
@@ -0,0 +1,23 @@
+# chage
+
+> Change user account and password expiry information.
+
+- List password information for the user:
+
+`chage -l {{user_name}}`
+
+- Enable password expiration in 10 days:
+
+`sudo chage -M {{10}} {{user_name}}`
+
+- Disable password expiration:
+
+`sudo chage -M 99999 {{user_name}}`
```

Comment:
The man page says that `Passing the number -1 as MAX_DAYS will remove checking a password's validity`. So, shouldn't it be -1 ?

Reply:
you are absolutely right, and I'll change that.
```
➜  ~ sudo chage -l test
Last password change                                    : Dec 05, 2017
Password expires                                        : never
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : 99999
Number of days of warning before password expires       : 7
➜  ~ sudo chage -M 10 test
➜  ~ sudo chage -l test   
Last password change                                    : Dec 05, 2017
Password expires                                        : Dec 15, 2017
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : 10
Number of days of warning before password expires       : 7
➜  ~ sudo chage -M -1 test
➜  ~ sudo chage -l test   
Last password change                                    : Dec 05, 2017
Password expires                                        : never
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : -1
Number of days of warning before password expires       : 7
```